packages/flutter/lib/src/services/message_codec.dart

```diff
@@ -169,6 +171,9 @@ class MissingPluginException implements Exception {
   /// A human-readable error message, possibly null.
   final String? message;
 
+  // `keepToString` to avoid removal of the toString implementation and help
+  // diagnose misbehaving plugins.
```

Comment:
nit: Maybe something like `// Keep the toString implementation so that we can more easily tell what plugin is missing even in release mode`?